easybuild/easyconfigs/i/iimkl/iimkl-2018a.eb

```diff
@@ -0,0 +1,20 @@
+# This is an easyconfig file for EasyBuild, see http://easybuilders.github.io/easybuild
+easyblock = 'Toolchain'
+
+name = 'iimkl'
```

Comment:
This might be somewhat confusing considering that iomkl is iccifort + OpenMPI + imkl

Should it be named ismkl for iccifort + serial + imkl ??

Reply:
That ship has sailed years ago, see https://github.com/easybuilders/easybuild-framework/pull/2082

Reply:
Ah, didn't look into that part...

Reply:
iimkl has already been merged in framework some time ago but it does follow some logic.
Adding an s is inconsistent with `golf` then too, which would then need to be called `gsolf`.

we'd had perfect alphabet soup if `imkl` would just be called `mkl` but alas.